File: src/main/java/com/smaboy/demo/entity/Response.java
package com.smaboy.demo.entity;

public class Response {
    String msg;
    int code;

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    @Override
    public String toString() {
        return "Response{" +
                "msg='" + msg + '\'' +
                ", code=" + code +
                '}';
    }
}

File: src/main/java/com/smaboy/demo/entity/Response.kt
package com.smaboy.demo.entity

/**
 * 类名: Response
 *
 * 类作用描述: java类作用描述
 *
 * 作者: liyongliang3
 *
 * 创建时间: 2020/4/16 7:04 下午
 *
 */
data class Response @JvmOverloads constructor(var code : Int = 0, var msg : String = "", var data : Any? = null)